public/app/features/trails/AutomaticMetricQueries/query-generators/histogram.ts

```diff
@@ -78,7 +69,9 @@ function percentileQuery(percentile: number, groupings: string[] = []) {
   const percent = percentile / 100;
 
   return {
-    refId: 'A',
+    // By including the specific refIds, the BreakdownScene can
+    // compare refIds to find a match
```

Comment:
I suppose this isn't necessary, now that I've removed the code that determines min and max from the main vizPanel queries.